tests/inspection/canceled/pass.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+
+set -e
+WAKE="${1:+$1/wake}"
+
+# Start from any empty db every time for stable job ids
+rm -f wake.db
+
+# Use || true to ignore the expected non-0 return from timeout
+timeout 1 ${WAKE} test || true
```

Comment:
We don't need `test Unit` here?

Reply:
Nope. `test` is exported as a CLI function so it expects a `List String` and with nothing is specified then `Nil` is passed